browser/admin/src/integrator/Xcu.ts

```diff
@@ -144,7 +144,7 @@ class Xcu {
 					: this.parse(XcuFileContent);
 		} catch (error) {
 			(window as any).SettingIframe.showErrorModal(
-				'Something went wrong while loading Document view settings.',
+				'Something went wrong while loading Document settings.',
```

Comment:
Error messages in this file do not seem localizable (no _('') markup!)

Reply:
I have created PR for xcu and workbook file : https://github.com/CollaboraOnline/online/pull/11766